.github/workflows/main.yml

```
@@ -0,0 +1,33 @@
+on:
+  push:
+  issues:
+    types:
+      - opened
+      - edited
+
+jobs:
+  create_calendar_from_issues:
+    runs-on: ubuntu-latest
+    name: Create calendar from issues
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create calendar
+        id: hello
+        uses: GBKS/github-events-calendar@v0.1
+        with:
+          domain: 'bitcoindesigners.org'
+          company: 'Bitcoin Design Community'
+          name: 'Bitcoin Design Community Calls & Events'
+          repository: "BitcoinDesign,Meta,call"
+          file: 'events.ical'
+      - name: Commit files
+        run: |
+          git config --local user.email "calendar-action@github.com"
+          git config --local user.name "iCal updater"
+          git add events.ical
+          git diff --quiet && git diff --staged --quiet || git commit -m "Update events.ical from issues"
```

This can be ignored 😅 I just can't resist the opportunity to shill conventional commits.

```suggestion
          git diff --quiet && git diff --staged --quiet || git commit -m "chore(calendar): update events.ical from opened/edited call issue"
```

---

What's a conventional commit?

---

Either way, sounds amazing, and I'm making the update.

---

haha I love using it. It's just a standard way to write commit messages that are machine readable, and also add some more context to commit messages for humans.

Benefits are
1. The strict structure makes it faster to come up with commit messages (at least for me)
2. Git logs are cleaner so easier to understand changes
3. Can allow scripts autogenerate versions and change logs